doc/manipulating_visualizing/data_preparation.rst

```diff
@@ -436,3 +435,29 @@ possible option.
 
    * :ref:`sphx_glr_auto_examples_connectivity_plot_probabilistic_atlas_extraction.py`
 
+Extraction of signals from seeds:\  :class:`NiftiSpheresMasker`.
+==================================================================
+
+The purpose of :class:`NiftiSpheresMasker` is to compute signals from
+seeds containing voxels in spheres. It makes it easy to get these signals once
+you have a list of coordinates.
+A single seed is a sphere defined by the radius (in millimeters) and the
+coordinates (typically MNI or TAL) of its center.
+
+Using :class:`NiftiSpheresMasker` needs to define a list of coordinates.
+"seeds" argument takes a list of 3D coordinates (tuples) of the spheres centers,
+they should be in the same space as the images.
+Seeds can overlap spatially and are represented in a binary present-nonpresent
+coding (no weighting).
+Below is an example of a coordinates list of four seeds from the default mode network.
+
+.. literalinclude:: ../../examples/connectivity/plot_adhd_spheres.py
+    :start-after: # Coordinates of Default Mode Network
+    :end-before: labels = [
+
+"radius" is an optional argument that takes a real value in millimeters.
+If no value is given the spheres are considered as voxels.
```

Comment:
I finally understood what you meant. I think that I would rephrase this as:

If no value is given for the radius argument, the single voxel at the given locus is used.